conf/cassandra.yaml

```diff
@@ -2368,3 +2368,104 @@ drop_compact_storage_enabled: false
+#      # If you want to avoid repairing nodes in one or more data centers, then you can specify the data centers in this list.
+#      ignore_dcs: []
+#      # Set this 'true' if AutoRepair should repair only the primary ranges owned by this node; else, 'false'.
+#      # It is the same as -pr in nodetool repair options.
+#      repair_primary_token_range_only: true
+#      # The number of nodes running repair parallelly. If parallel_repair_percentage is set, it will choose the larger value of the two.
+#      # This configuration controls how many nodes would run repair in parallel.
+#      # The value “3” means, at any given point in time, at most 3 nodes would be running repair in parallel. These selected nodes can be from any datacenters.
+#      # If one or more node(s) finish repair, then the framework automatically picks up the next candidate(s) based on the least repair time.
+#      # It will ensure the maximum number of nodes running repair do not exceed “3”.
+#      parallel_repair_count: 3
+#      # The percentage of nodes in the cluster that run repair parallelly. If parallel_repair_count is set, it will choose the larger value of the two.
+#      # The problem with a fixed number of nodes (parallel_repair_count property) is that in a large-scale environment,
+#      # the nodes keep getting added/removed due to elasticity, so if we have a fixed number, then manual interventions would increase because, on a continuous basis,operators would have to adjust to meet the SLA.
+#      # The default is 3%, which means that 3% of the nodes in the Cassandra cluster would be repaired in parallel.
+#      # So now, if a fleet, an operator won't have to worry about changing the repair frequency, etc., as overall repair time will continue to remain the same even if nodes are added or removed due to elasticity.
+#      # Extremely fewer manual interventions as it will rarely violate the repair SLA for customers.
+#      parallel_repair_percentage: 3
+#      # If the scheduler should repair MV table or not.
+#      mv_repair_enabled: false
+#      # After a node restart, wait for this much delay before scheduler starts running repair; this is to avoid starting repair immediately after a node restart.
+#      initial_scheduler_delay: 5m
+#      # The major issue with Repair is sometimes repair session hangs; so this timeout is useful to resume such stuck repair sessions.
+#      repair_session_timeout: 3h
+#      # Whether to force immediate repair on new nodes. This is useful if you want to repair newly bootstrapped nodes immediately after they join the ring.
+#      force_repair_new_node: false
+#      # Splitter implementation to use for generating repair assignments.
+#      # The default is {@link RepairTokenRangeSplitter}. The class should implement {@link IAutoRepairTokenRangeSplitter}
```

Comment:
Replace `{@link RepairTokenRangeSplitter}` with just `RepairTokenRangeSplitter` since the former only works with javadoc.

Reply:
Done